### DISCUSSION TITLE
Hono instead of Rutt?

### DISCUSSION THREAD

**Discussion by sant123** [freshframework/fresh#743] [Ideas]
I'm curious if Hono would be a good fit for Fresh since currently is the fastest router?

Thanks.

**Comment by sant123** [comment]
🤔 

https://github.com/denosaurs/bench/commit/9600c2f5c88ff8622e91cacada661278b55b5aa2

**Comment by marvinhagemeister** [comment]
Hono is a great framework. When it comes to Fresh it doesn't make sense to build on top of Hono as we have some Fresh-specific routing features like route groups. Having to jump repositories for new features only increases maintenance burden for us, so we'd like to keep as much as possible in this one.
